### PR TITLE
Add section on Client identity

### LINKED ISSUE #49: What stops a client from using a ton of mediators?
I may have a good default mediator that uses device attestation, etc. However, if the set of mediators can grow in an unbounded fashion and be per-account or something, then clients could try to use as many mediators as possible to get more access. Limiting the overall number of mediators has other concerns.

### LINKED ISSUE #70: Discuss client identity
It's not limited by this document, but we should explain the notion of client identity, which is based on how mediators view clients. We should talk about expectations.

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #70.
Closes #49.

### CODE REVIEW

File: draft-private-access-tokens.md [hunk 1]
```diff
@@ -274,6 +274,49 @@ for more details. A Client can use a service offered by its Mediator or a
 third-party to store these secrets, but it is a requirement of the PAT
 architecture that the Mediator not be able to learn these secrets.
 
+## Client Identity
+
+The PAT architecture does not enforce strong constraints around the definition
+of a Client identity and allows it to be defined entirely by a Mediator. If a
+user can create an arbitrary number of Client identities that are accepted by
+one or more Mediators, a malicious user can easily game this architecture and
```
Comment (chris-wood) [inline draft-private-access-tokens.md:282]:
```suggestion
one or more Mediators, a malicious user can easily abuse the system to
```

File: draft-private-access-tokens.md [hunk 2]
```diff
@@ -274,6 +274,49 @@ for more details. A Client can use a service offered by its Mediator or a
...
+
+These multiple identities could be Sybil or true identities.
+
+A Mediator alone is responsible for detecting and weeding out Sybil Client
+identities in the PAT architecture. An Issuer relies on a Mediator's reputation;
+as explained in {{properties}}, the correctness of the architecture hinges on
+Issuers issuing tokens to only known and reputable Mediators.
+
+Users have multiple true identities on the Internet however, and as a result, it
+seems possible for a user to game this architecture without having to create
```
Comment (chris-wood) [inline draft-private-access-tokens.md:293]:
```suggestion
seems possible for a user to abuse the system without having to create
```

File: draft-private-access-tokens.md [hunk 3]
```diff
@@ -274,6 +274,49 @@ for more details. A Client can use a service offered by its Mediator or a
...
+- An Issuer can learn the properties of a Mediator - specifically, which stable
+  Client identifier is authenticated by the Mediator - to determine whether the
+  Mediator is acceptable for an Origin.
+
+- An Origin can choose an Issuer based on the types of Mediators accepted by the
+  Issuer, or the Origin can communicate its constraints to the designated
+  Issuer.
+
+- An Origin can direct a user to a specific Issuer based on client properties
+  that are visible. For instance,
```
Comment (chris-wood) [inline draft-private-access-tokens.md:310]:
```suggestion
  that are visible. For instance, properties that are observable in the HTTP User Agent string.
```
I wasn't sure what you had in mind, but maybe this works?

File: draft-private-access-tokens.md (hunk 1 above)
Comment (tfpauly) [reply draft-private-access-tokens.md:282]: I like this suggestion

File: draft-private-access-tokens.md [hunk 4]
```diff
@@ -274,6 +274,49 @@ for more details. A Client can use a service offered by its Mediator or a
...
 
+## Client Identity
+
+The PAT architecture does not enforce strong constraints around the definition
+of a Client identity and allows it to be defined entirely by a Mediator. If a
+user can create an arbitrary number of Client identities that are accepted by
+one or more Mediators, a malicious user can easily game this architecture and
+defeat the Issuer's ability to enforce per-Client policies.
+
+These multiple identities could be Sybil or true identities.
```
Comment (tfpauly) [inline draft-private-access-tokens.md:285]: Sybil?

**Pull request merged**